### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by samvloeberghs**
https://github.com/AngularClass/angular2-webpack-starter/compare/master...samvloeberghs:master#diff-04c6e90faac2675aa89e2176d2eec7d8L49:

When you follow the guide and you install typings globally, you don't need to reference it into node modules, right?

https://github.com/AngularClass/angular2-webpack-starter/compare/master...samvloeberghs:master#diff-04c6e90faac2675aa89e2176d2eec7d8L54

I just followed you guys setup description and http://0.0.0.0:3000 doesn't work by default on a Windows machine. So I've added http://localhost:3000.

Grtz

### CODE REVIEW

**Comment by samvloeberghs** [comment]
I've just spotted the use of 0.0.0.0 in the webpack config files as well.
https://github.com/AngularClass/angular2-webpack-starter/blob/master/webpack.prod.config.js#L19
https://github.com/AngularClass/angular2-webpack-starter/blob/master/webpack.config.js#L18

Is it ok to just use localhost? On a Windows machine, the use of 0.0.0.0 is breaking functionality hard.

**Comment by gdi2290** [comment]
`0.0.0.0` was used for virtual machines  but we can change it back and I'll just add it to FAQ.

File: README.md [hunk 1]
```diff
@@ -46,12 +46,12 @@ cd angular2-webpack-starter
 npm install
 
 # install TypeScript typings
-./node_modules/.bin/typings install
```
Comment (gdi2290) [inline README.md]: can you leave this

File: README.md [hunk 2]
```diff
@@ -46,12 +46,12 @@ cd angular2-webpack-starter
...
+typings install
 
 # start the server
 npm start
 ```
-go to [http://0.0.0.0:3000](http://0.0.0.0:3000) in your browser
+go to [http://0.0.0.0:3000](http://0.0.0.0:3000) or [http://localhost:3000](http://localhost:3000) in your browser
```
Comment (gdi2290) [inline README.md]: can you also update the webpack.config

**Comment by samvloeberghs** [comment]
done but please validate

**Comment by gdi2290** [comment]
thanks!

**Pull request merged**